projects/angular/src/popover/tooltip/API.md

```diff
@@ -0,0 +1,26 @@
+# tooltip-clarity - Style API
+
+
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-tooltip-background-color | Changes the tooltip's background color and is applied to all tooltip elements.
+| --clr-tooltip-border-radius | Changes the tooltip element's corner radius and is applied to all tooltip elements.
+|  --clr-tooltip-color | Changes the color of the text inside a tooltip element. This change affects all tooltip elements.
+| --clr-tooltip-font-weight | changes the thickness of text characters and is applied to all text contained within tooltip elements
+
+
+## Class names
+
+| Class name    | Description                              |
+| ------------- | ---------------------------------------- |
+| clr-tooltip | The tooltip class includes tooltip elements. When the user moves the mouse pointer over an tooltip element, extra information about that element is specified. |
```

Comment:
Prefer to use "hovers" instead of move mouse

Reply:
Updated, As suggested